source/renderer/app/api/api.js

```diff
@@ -635,7 +637,16 @@ export default class AdaApi {
       logger.debug('AdaApi::getAssets success', {
         assets: response,
       });
-      const assets = response.map((asset) => _createAssetFromServerData(asset));
+      const assetsLocaldata = await global.daedalus.api.localStorage.getAssetsLocalData();
+      logger.debug('AdaApi::getAssetsLocalData success', {
+        assetsLocaldata,
+      });
+      const assets = response.map((asset) =>
+        _createAssetFromServerData(
+          asset,
+          assetsLocaldata[asset.fingerprint] || {}
```

Comment:
@daniloprates we should be fetching the data by `policyId+assetName` and not `fingerprint`.